ui/src/locales/en.json

```diff
@@ -0,0 +1,55 @@
+{
+    "alertFeedback": {
+        "info": "Information",
+        "success": "Success",
+        "warning": "Warning",
+        "danger": "Danger"
+    },
+    "faq": {
+        "whatIsZim": "What is a Zim file?",
+        "whatIsZimDesc": "The Zim file format stores website content for offline usage. It assembles the normal constituent of a website into a single archive, and compresses it so as to make it easier to save, share, and store.",
+        "howToRead": "How do I read my Zim files?",
+        "howToReadDesc": "You will need a Zim file reader. This usually means Kiwix, which is available on desktop computers, mobile devices, and more. Currently only Kiwix-serve and Kiwix-Android can read all Zimit-generated files. If using Kiwix-Desktop for Microsoft Windows and GNU/Linux, then you will need to configure it as a Kiwix-serve instance in the settings. We expect most platforms to be upgraded by the end of 2021.",
+        "missingContent": "The Zim file is incomplete or smaller than the original website",
+        "missingContentDesc": "Because of the very nature of this tool, we can’t leave it open for unlimited requests towards any website. That could be harmful both for our infrastructure, but also for the target websites. We currently enforce two limits: file size and time.",
+        "gotError": "I got an error message (no zim) or could not read a zim file",
+        "gotErrorDesc": "Triple-check the URL you entered, and if it is still not working then open a bug ticket on github. Indicate the target website, the request number (it’s in the email you received), and the device you tried to open your zim file on."
+    },
+    "loading": {
+        "loading": "Loading…"
+    },
+    "navbar": {
+        "logoAlt": "Youzim.it logo"
```

Comment:
This has not been used in code while it should

Reply:
I will push the commits with these changes,